src/screens/doc/DocIconsScreen.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { FlatList, StyleSheet, View } from 'react-native';
+
+import { RegularText } from '../../components';
+import { colors, Icon, normalize } from '../../config';
+
+type TIconItem = {
+  iconName: keyof typeof Icon;
+};
+
+const IconItem = ({ iconName }: TIconItem) => {
+  const SelectedIcon = Icon[iconName];
+
+  return (
+    <View style={styles.iconContainer}>
+      <SelectedIcon size={normalize(35)} color={colors.secondary} />
```

Comment:
there is an ts problem here. did you try to resolve it?

Reply:
yes i tried to but have not found a solution yet.

Reply:
ok, then please add a ts lint ignore comment for that

Reply:
managed to solve it, added  name={'link'} to SelectedIcon